### PR TITLE
[release/public-v2.2.0]: Documentation Updates

### LINKED ISSUE #949: Update documentation for v2.2.0 release
## Description
Add a variety of updates required for the v2.2.0 release, including: 
- Information on changes since v2.1.0 release
- Update all hyperlinks, data paths, git clone command, container commands, and version numbers to reflect content of release
- Update LAM grid chapter to include RRFS_NA_13km grid
- Update DOI/citation in docs & README
- Update VX chapter w/new release info

## Solution
Merge these updates into the appropriate RST files in the release branch.

### PR DESCRIPTION
**Pull request by gspetro-NOAA**
## DESCRIPTION OF CHANGES: 
Implements changes in Issue #949. 

### Type of change
- [X] This change requires a documentation update

## TESTS CONDUCTED: 
None required. The human-readable version of my fork docs can be viewed here: https://srw-ug.readthedocs.io/en/text-release-updates/

- [ ] hera.intel
- [ ] orion.intel
- [ ] hercules.intel
- [ ] cheyenne.intel
- [ ] cheyenne.gnu
- [ ] derecho.intel
- [ ] gaea.intel
- [ ] gaeac5.intel
- [ ] jet.intel
- [ ] wcoss2.intel
- [ ] NOAA Cloud (indicate which platform)
- [ ] Jenkins
- [ ] fundamental test suite
- [ ] comprehensive tests (specify *which* if a subset was used)

## DEPENDENCIES:
None.

## DOCUMENTATION:
N/A. All documentation.

## ISSUE: 
Resolves Issue #949 . 

## CHECKLIST

- [ ] My code follows the style guidelines in the Contributor's Guide
- [ ] I have performed a self-review of my own code using the Code Reviewer's Guide
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes need updates to the documentation. I have made corresponding changes to the documentation
- [ ] My changes do not require updates to the documentation (explain).
- [ ] My changes generate no new warnings
- [ ] New and existing tests pass with my changes
- [ ] Any dependent changes have been merged and published

## LABELS (optional): 

A Code Manager needs to add the following labels to this PR: 
- [ ] Work In Progress
- [ ] bug
- [ ] enhancement
- [ ] documentation
- [ ] release
- [ ] high priority
- [ ] run_ci
- [ ] run_we2e_fundamental_tests
- [ ] run_we2e_comprehensive_tests
- [ ] Needs Cheyenne test 
- [ ] Needs Jet test 
- [ ] Needs Hera test 
- [ ] Needs Orion test 
- [ ] help wanted

### CODE REVIEW

File: docs/UsersGuide/source/BuildingRunningTesting/VXCases.rst
Comment (gspetro-NOAA) [inline docs/UsersGuide/source/BuildingRunningTesting/VXCases.rst]: @EdwardSnyder-NOAA I will need your feedback or a copy of your config file to ensure that the changes to this config.yaml excerpt are correct and complete.

File: docs/UsersGuide/source/BackgroundInfo/Introduction.rst
Comment (MichaelLueken) [inline docs/UsersGuide/source/BackgroundInfo/Introduction.rst:14]: Support was also added for the `FV3_GFS_v17_p8` physics suite (from PR #574).

**Review by MichaelLueken — COMMENTED** [review]
@gspetro-NOAA - I've gone through the updates since v2.1 in the Introduction chapter and the only thing I noticed as missing was the support of the `FV3_GFS_v17_p8` CCPP physics suite, which was brought in with PR #574.

File: docs/UsersGuide/source/BackgroundInfo/Introduction.rst
Comment (gspetro-NOAA) [reply docs/UsersGuide/source/BackgroundInfo/Introduction.rst:14]: Is `FV3_GFS_v17_p8` also considered fully supported? If so, which grids does it play nice with? (I'd have to document that) 
@mkavulich feel free to chime in ;)

Comment (MichaelLueken) [reply docs/UsersGuide/source/BackgroundInfo/Introduction.rst:14]: I think what you have done in hash [0d02479](https://github.com/ufs-community/ufs-srweather-app/pull/950/commits/0d02479f116665dde3dd91b44f66e9cb32f6ca2c) is sufficient.  This physics suite was added because it was being used in the weather model, but not in the SRW App.  However, I'd let others make the final decision on whether this physics suite is fully supported, or just added to ensure that we are up to date with respect to the physics suites that are used in the weather model.

File: docs/UsersGuide/source/BuildingRunningTesting/VXCases.rst
Comment (EdwardSnyder-NOAA) [reply docs/UsersGuide/source/BuildingRunningTesting/VXCases.rst]: Three changes to the config file:

-  Remove the `NOHRSC_OBS_DIR` variable because we don't use these obs for this case.
- Change the physics suite variable to RRFS beta1 i.e. `CCPP_PHYS_SUITE: FV3_RRFS_v1beta`. The reason being is that this physics suite better matches the grid we are using.
- Change the verification name i.e. `VX_FCST_MODEL_NAME: FV3_RRFS_v1beta_SUBCONUS_Ind_3km`. Making this change means that the vx files mentioned throughout this document needs to change as well. See my other comment.

Here is the config file I used to run the experiment:
```
metadata:
  description: >-
    Sample community config
user:
  RUN_ENVIR: community
  MACHINE: hera
  ACCOUNT: epic
platform:
  CCPA_OBS_DIR: "/scratch1/NCEPDEV/nems/Edward.Snyder/indy-srw-sample-vx-2p2/Indy-Severe-Weather/obs_data/ccpa/proc"
  MRMS_OBS_DIR: "/scratch1/NCEPDEV/nems/Edward.Snyder/indy-srw-sample-vx-2p2/Indy-Severe-Weather/obs_data/mrms/proc"
  NDAS_OBS_DIR: "/scratch1/NCEPDEV/nems/Edward.Snyder/indy-srw-sample-vx-2p2/Indy-Severe-Weather/obs_data/ndas/proc"
workflow:
  USE_CRON_TO_RELAUNCH: false
  EXPT_SUBDIR: test_community
  CCPP_PHYS_SUITE: FV3_RRFS_v1beta
  PREDEF_GRID_NAME: SUBCONUS_Ind_3km
  DATE_FIRST_CYCL: '2019061500'
  DATE_LAST_CYCL: '2019061500'
  FCST_LEN_HRS: 60
  PREEXISTING_DIR_METHOD: rename
  VERBOSE: true
  COMPILER: intel
task_get_extrn_ics:
  EXTRN_MDL_NAME_ICS: FV3GFS
  FV3GFS_FILE_FMT_ICS: grib2
  EXTRN_MDL_SOURCE_BASEDIR_ICS: /scratch1/NCEPDEV/nems/Edward.Snyder/indy-srw-sample-vx-2p2/Indy-Severe-Weather/input_model_data/FV3GFS/grib2/2019061500
  USE_USER_STAGED_EXTRN_FILES: true
task_get_extrn_lbcs:
  EXTRN_MDL_NAME_LBCS: FV3GFS
  LBC_SPEC_INTVL_HRS: 6
  FV3GFS_FILE_FMT_LBCS: grib2
  EXTRN_MDL_SOURCE_BASEDIR_LBCS: /scratch1/NCEPDEV/nems/Edward.Snyder/indy-srw-sample-vx-2p2/Indy-Severe-Weather/input_model_data/FV3GFS/grib2/2019061500
  USE_USER_STAGED_EXTRN_FILES: true
task_run_fcst:
  QUILTING: true
task_plot_allvars:
  COMOUT_REF: ""
global:
  DO_ENSEMBLE: false
  NUM_ENS_MEMBERS: 2
verification:
  VX_FCST_MODEL_NAME: FV3_RRFS_v1beta_SUBCONUS_Ind_3km
rocoto:
  tasks:
    metatask_run_ensemble:
      task_run_fcst_mem#mem#:
        walltime: 02:00:00
    taskgroups: '{{ ["parm/wflow/prep.yaml", "parm/wflow/coldstart.yaml", "parm/wflow/post.yaml", "parm/wflow/verify_pre.yaml", "parm/wflow/verify_det.yaml"]|include }}'
```

File: docs/UsersGuide/source/BuildingRunningTesting/VXCases.rst
Comment (EdwardSnyder-NOAA) [inline docs/UsersGuide/source/BuildingRunningTesting/VXCases.rst:1]: The files `point_stat_FV3_GFS_v16_SUBCONUS_3km_NDAS_ADPSFC_300000L_20190616_060000V.stat` and  `grid_stat_FV3_GFS_v16_SUBCONUS_3km_NDAS_ADPSFC_300000L_20190616_060000V.stat` names would change to  `point_stat_FV3_RRFS_v1beta_SUBCONUS_3km_NDAS_ADPSFC_300000L_20190616_060000V.stat` and `grid_stat_FV3_RRFS_v1beta_SUBCONUS_3km_NDAS_ADPSFC_300000L_20190616_060000V.stat`.

File: docs/UsersGuide/source/BackgroundInfo/Introduction.rst
Comment (natalie-perlin) [inline docs/UsersGuide/source/BackgroundInfo/Introduction.rst]: @gspetro-NOAA - this is a great summary!

Comment (gspetro-NOAA) [reply docs/UsersGuide/source/BackgroundInfo/Introduction.rst]: Thanks! :)

File: docs/UsersGuide/source/BuildingRunningTesting/VXCases.rst
Comment (gspetro-NOAA) [reply docs/UsersGuide/source/BuildingRunningTesting/VXCases.rst:1]: For `grid_stat_*`/`point_stat_*` files, I mostly have templates, like `grid_stat_FV3_GFS_v16_SUBCONUS_3km_REFC_MRMS_HHMMSSL_YYYYMMDD_HHMMSSV.stat`. I don't see a `grid_stat_FV3_GFS_v16_SUBCONUS_3km_NDAS_ADPSFC_300000L_20190616_060000V.stat` file in the docs, so let me know if/where it needs adding. Seems like the main difference is that the physics suite on all of the files needs adjusting from FV3_GFS_v16 to FV3_RRFS_v1beta, so that's what I did. Let me know if I need to fix anything else! :)

Comment (EdwardSnyder-NOAA) [reply docs/UsersGuide/source/BuildingRunningTesting/VXCases.rst:1]: I had a copy+paste error in this comment. It should be the full name of the physics suite plus the grid. We are missing Ind in the file name. They should be: <type>_stat_FV3_RRFS_v1beta_SUBCONUS_Ind_3km_<product>_....

Comment (EdwardSnyder-NOAA) [reply docs/UsersGuide/source/BuildingRunningTesting/VXCases.rst:1]: Sorry again for the typo. NDAS is a point stat and not a grid. Just update the physics suite from GFS to RRFS, and add Ind to the filename.

Comment (gspetro-NOAA) [reply docs/UsersGuide/source/BuildingRunningTesting/VXCases.rst:1]: Np! Made the change. Let me know if I missed anything.

File: docs/UsersGuide/source/BuildingRunningTesting/VXCases.rst
Comment (EdwardSnyder-NOAA) [inline docs/UsersGuide/source/BuildingRunningTesting/VXCases.rst]: The link here talks about `Plotting Configuration`, which is fine, but doesn't show the user how to run the plotting task. Since the last release, the plotting task was integrated into the workflow, so I'd suggest adding this section in with the `Configure the Verification Sample Case` section. Add `"parm/wflow/plot.yaml"` to the task group, and add the code block mentioned here (`task_plot_allvars`) to the config section. 

Or if we don't want to merge it into `Configure the Verification Sample Case` section, then we should at least move it ahead of the `Generate the Experiment`, because it is hard to run plotting once the experiment is generated. You'll have to rerun the generation script again.

File: docs/UsersGuide/source/BuildingRunningTesting/Quickstart.rst
Comment (EdwardSnyder-NOAA) [inline docs/UsersGuide/source/BuildingRunningTesting/Quickstart.rst]: There is a random space in this path. Replace it with a '/' will return a full path to the grib2 files on Gaea. Same with the LBCS path.

Comment (gspetro-NOAA) [reply docs/UsersGuide/source/BuildingRunningTesting/Quickstart.rst]: Fixed!

File: docs/UsersGuide/source/BuildingRunningTesting/WE2Etests.rst
Comment (MichaelLueken) [inline docs/UsersGuide/source/BuildingRunningTesting/WE2Etests.rst:176]: In the `Tests.csv` file:
1. The `grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_2017_gfdlmp` test has been changed to `grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_RAP`.  The `CCPP_PHYS_SUITE` has been changed from `FV3_GFS_2017_gfdlmp` to `FV3_RAP`.
2. The `grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16` test has been changed to `grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16_plot`.  No other changes, outside of the name of the test, need to be made.

Outside of these modifications to two tests, it looks like the rest of the supported tests are up-to-date.

File: docs/UsersGuide/source/BackgroundInfo/Introduction.rst
Comment (MichaelLueken) [inline docs/UsersGuide/source/BackgroundInfo/Introduction.rst]: I believe that this is a good list indicating what we will be focussing on for future releases.  The only potential minor addition would be noting that JEDI will be the DA system that we will be including.  With the GSI already available, the community might wonder why the GSI hasn't been added to the SRW App.  Since we are unable to release the GSI to the community, clearing this up might be a good idea.

**Review by MichaelLueken — COMMENTED** [review]
@gspetro-NOAA - I've reviewed the list of supported WE2E tests in the `Tests.csv` file and noted two tests that have been altered.  I've also suggested adding JEDI to the DA entry in the introduction for the list of updates planned for the future (to explain why the GSI isn't being added to the SRW App).

File: docs/UsersGuide/source/BuildingRunningTesting/RunSRW.rst
Comment (EdwardSnyder-NOAA) [inline docs/UsersGuide/source/BuildingRunningTesting/RunSRW.rst]: This should be: `/path/to/UFS_SRW_data/v2p2/input_model_data/<model_type>/<data_type>/${yyyymmddhh}`
Awhile back we changed our staged data location from `UFS_SRW_App` to `UFS_SRW_data`.

This goes for the `EXTRN_MDL_SOURCE_BASEDIR_LBCS` variable as well.

File: docs/UsersGuide/source/BuildingRunningTesting/RunSRW.rst
Comment (EdwardSnyder-NOAA) [inline docs/UsersGuide/source/BuildingRunningTesting/RunSRW.rst]: All these variables should be `/path/to/UFS_SRW_data/...`

File: docs/UsersGuide/source/BuildingRunningTesting/RunSRW.rst
Comment (EdwardSnyder-NOAA) [inline docs/UsersGuide/source/BuildingRunningTesting/RunSRW.rst]: This file was removed since it is a RRFS task.

File: docs/UsersGuide/source/BuildingRunningTesting/RunSRW.rst
Comment (gspetro-NOAA) [inline docs/UsersGuide/source/BuildingRunningTesting/RunSRW.rst:71]: @MichaelLueken @EdwardSnyder-NOAA Does the WCOSS location need to be updated to `UFS_SRW_data`?

Comment (MichaelLueken) [reply docs/UsersGuide/source/BuildingRunningTesting/RunSRW.rst:71]: Unfortunately, WCOSS2 is only really accessible by EMC folks.

@chan-hoo @BenjaminBlake-NOAA @MatthewPyle-NOAA - Is the provided path,  `/lfs/h2/emc/lam/noscrub/UFS_SRW_App/develop/input_model_data/`, correct for staged data on WCOSS2?  We are wanting to make sure before moving forward and merging this work to the SRW v2.2 release branch.  Thanks!

File: docs/UsersGuide/source/BackgroundInfo/Introduction.rst
Comment (gspetro-NOAA) [reply docs/UsersGuide/source/BackgroundInfo/Introduction.rst]: @MichaelLueken Added "via JEDI" to the bullet about DA/cycling. Should I explicitly mention why we are not including GSI, or is it enough to note that we will be including DA via JEDI?

Comment (MichaelLueken) [reply docs/UsersGuide/source/BackgroundInfo/Introduction.rst]: @gspetro-NOAA Thanks!  I think adding "via JEDI" should be sufficient for DA/cycling.

File: docs/UsersGuide/source/BuildingRunningTesting/RunSRW.rst
Comment (chan-hoo) [reply docs/UsersGuide/source/BuildingRunningTesting/RunSRW.rst:71]: @MichaelLueken, I'll check out and test this release branch on WCOSS2, and let you know soon.

Comment (MichaelLueken) [reply docs/UsersGuide/source/BuildingRunningTesting/RunSRW.rst:71]: Many thanks to @MatthewPyle-NOAA for providing the following:
```
Looks like it still has it there on WCOSS

ls /lfs/h2/emc/lam/noscrub/UFS_SRW_App/develop/input_model_data/
FV3GFS  GSMGFS  HRRR  NAM  RAP
```
So, `/lfs/h2/emc/lam/noscrub/UFS_SRW_App/develop/input_model_data/` is still the correct location on WCOSS2.

Comment (chan-hoo) [reply docs/UsersGuide/source/BuildingRunningTesting/RunSRW.rst:71]: @MichaelLueken, one we2e test has been completed successfully on WCOSS2 as follows:
```
       CYCLE                    TASK                       JOBID               STATE         EXIT STATUS     TRIES      DURATION
================================================================================================================================
201907010000               make_grid                   107737134           SUCCEEDED                   0         1          15.0
201907010000               make_orog                   107737172           SUCCEEDED                   0         1          42.0
201907010000          make_sfc_climo                   107737203           SUCCEEDED                   0         1          68.0
201907010000           get_extrn_ics                   107737135           SUCCEEDED                   0         1          11.0
201907010000          get_extrn_lbcs                   107737136           SUCCEEDED                   0         1          11.0
201907010000         make_ics_mem000                   107737305           SUCCEEDED                   0         1         127.0
201907010000        make_lbcs_mem000                   107737306           SUCCEEDED                   0         1         210.0
201907010000         run_fcst_mem000                   107737514           SUCCEEDED                   0         1         237.0


Summary of workflow status:
~~~~~~~~~~~~~~~~~~~~~~~~~~

  1 out of 1 cycles completed.
  Workflow status:  SUCCESS

========================================================================
End of output from script "launch_FV3LAM_wflow.sh".
========================================================================


The end-to-end run of the workflow for the forecast experiment specified
by expt_name has completed with the following workflow status (wflow_status):
  expt_name = "grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v15p2"
  wflow_status = "SUCCESS"
```
As @MatthewPyle-NOAA mentioned above, the path to the input data is correct.

Comment (MichaelLueken) [reply docs/UsersGuide/source/BuildingRunningTesting/RunSRW.rst:71]: Thank you very much for the confirmation that the WE2E test works on WCOSS2 and that the path to the input data is correct, @chan-hoo!  Resolving this conversation now.

File: docs/UsersGuide/source/BackgroundInfo/Introduction.rst
Comment (MichaelLueken) [reply docs/UsersGuide/source/BackgroundInfo/Introduction.rst:14]: I will go ahead and resolve this conversation now.

**Review by EdwardSnyder-NOAA — APPROVED** [review]

**Review by MichaelLueken — APPROVED** [review]
@gspetro-NOAA - The modifications that I have requested have all been made.  These changes look good to me, so I will now give my approval.

**Pull request merged**
